File: src/main/java/pl/training/workshop/m7/s07_arrowhead/BookingAttempt.java
package pl.training.workshop.m7.s07_arrowhead;

/** Stabilny kontrakt sceny: próba rezerwacji z wynikami wcześniejszych sprawdzeń. */
public record BookingAttempt(String email, boolean screeningFound, boolean salesOpen,
        boolean customerBlocked, int requestedSeats, int freeSeats) {
}

File: csharp/src/Training.Workshop/M7/S07Arrowhead/BookingAttempt.cs
namespace Training.Workshop.M7.S07Arrowhead;

/// <summary>Stabilny kontrakt sceny: próba rezerwacji z wynikami wcześniejszych sprawdzeń.</summary>
public sealed record BookingAttempt(string Email, bool ScreeningFound, bool SalesOpen,
    bool CustomerBlocked, int RequestedSeats, int FreeSeats);
